server.go

```diff
@@ -302,11 +299,11 @@ func newNotebookServer(config string) (*notebookServer, error) {
 
 	srv.Handler = srv.mux
 
-	srv.tlsCert = sc.TLSCert
-	srv.tlsKey = sc.TLSKey
-	srv.httpRedirect = sc.HTTPRedirect
+	srv.tlsCert = srv.TLSCert
+	srv.tlsKey = srv.TLSKey
+	srv.httpRedirect = srv.HTTPRedirect
 
```

Comment:
Duplicates now, we should remove the private members (tlsCert, tlsKey, etc.), and fix when they are used.

Reply:
Basically use the config version only...

Reply:
Done, and Done